larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -146,6 +167,18 @@ void* Init(TfLiteContext* context, const char* buffer, size_t length) {
     return conv_params;
   }
 
+  if (conv_params->padding_type == TfLitePadding::kTfLitePaddingSame &&
+      conv_params->pad_value == 0 &&
```

Comment:
Should we check the following instead?
```suggestion
      conv_params->pad_value != 1 &&
```

Reply:
Yes that is cleaner. Right now its still fine because in `Init` it is asserted that the pad_value can only be 0 or 1.